backbone.js

```diff
@@ -440,6 +440,14 @@
       return _.clone(this._previousAttributes);
     },
 
+    // Returns `true` if all attributes is equal to model attributes
+    equal: function (attrs) {
+      for (var key in attrs) {
+        if (attrs[key] !== this.get(key)) return false;
```

Comment:
This isn't sufficiently equal anyway, its possible that `this.attributes` has more attributes in it still